Sources/ConfidenceProvider/ConfidenceFeatureProvider.swift

```diff
@@ -85,11 +86,12 @@ public class ConfidenceFeatureProvider: FeatureProvider {
 
         Task {
             do {
-                let resolveResult = try await resolve(context: initialContext)
+                let context = confidence?.getContext() ?? ConfidenceTypeMapper.from(ctx: initialContext)
```

Comment:
Does this assume that `confidence` already has processed/added `initialContext`? 
In a way I'd think it's more readable if we had something like 
```
let context = confidence?.getContext().merge(ConfidenceTypeMapper.from(ctx: initialContext))
```

Reply:
Oh yes, we call `self.updateConfidenceContext(context: initialContext)` a few lines above, never mind